lib/src/screen/dice/dice_screen.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+
+import '../../utils/constants.dart';
+
+class DiceScreen extends StatefulWidget {
+  DiceScreen({
+    Key key,
+    this.title,
+  }) : super(key: key);
+
+  final String title;
+
+  @override
+  _DiceScreenState createState() => _DiceScreenState();
+}
+
+class _DiceScreenState extends State<DiceScreen> {
+  @override
+  Widget build(
+    BuildContext context,
+  ) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(
+          widget.title,
+        ),
+      ),
+      body: GridView.count(
+        crossAxisCount: 2,
+        padding: EdgeInsets.all(25.0),
+        mainAxisSpacing: 15.0,
+        crossAxisSpacing: 40.0,
+        children: List.generate(
+          Constants.diceAmount,
+          (int index) {
+            return _createDiceContainer(index + 1);
+          },
+        ),
+      ),
+      bottomNavigationBar: Row(
+        children: [
+          SizedBox(
+            width: 10.0,
+          ),
+          Text(
+            'Tap on any dice to make it roll!',
+            style: TextStyle(
+              fontSize: 17.0,
+              fontWeight: FontWeight.w700,
+              height: Constants.textHeight1_4,
+            ),
+          ),
+          SizedBox(
+            width: 20.0,
+          ),
+          Text(
+            'Score: ',
+            style: TextStyle(
+              fontSize: 20.0,
+              fontWeight: Constants.fontWeight400,
+              height: Constants.textHeight1_4,
+            ),
+          ),
+          Text(
+            '21',
+            style: TextStyle(
+              fontSize: 22.0,
+              fontWeight: Constants.fontWeight400,
+              height: Constants.textHeight1_4,
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _createDiceContainer(int diceNumber) {
+    return Container(
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(25.0),
+        color: Colors.blueGrey[700],
+      ),
+      child: _diceValues(
+        diceNumber,
+      ),
+    );
+  }
+
+  Widget _diceValues(int diceFaces) {
+    if (diceFaces < 1 || diceFaces > 6) {
+      return Container();
+    }
```

Comment:
This condition is not necessary since you always pass a correct value as a parameter.

Reply:
If you delete this condition, the function is no longer necessary, you can move the creation of the image as a child inside the Container of the function _createDiceContainer

Reply:
Done